ambry-store/src/main/java/com.github.ambry.store/CompactionManager.java

```diff
@@ -63,8 +71,15 @@
   void enable() {
     if (compactionExecutor != null) {
       logger.info("Compaction thread started for {}", mountPath);
-      compactionThread = Utils.newThread("CompactionThread-" + mountPath, compactionExecutor, true);
+      compactionThread = Utils.newThread(THREAD_NAME_PREFIX + mountPath, compactionExecutor, true);
       compactionThread.start();
+      try {
+        if (!compactionExecutor.startLatch.await(1, TimeUnit.SECONDS)) {
```

Comment:
why do you care about this? This is essentially checking if the JVM started the thread we asked it to start, isn't it? That seems like an arbitrary check to do.

If this is being done for tests, perhaps you can use `TestUtils.getThreadByThisName()`?